gpkit/model.py

```diff
@@ -284,8 +286,9 @@ def _solve(self, programType, solver, verbosity, skipfailures,
         ValueError if programType and model constraints don't match.
         RuntimeWarning if an error occurs in solving or parsing the solution.
         """
-        signomials, unsubbed, allsubs = self.signomials_et_al
-        sweep, linkedsweep, constants = separate_subs(unsubbed, allsubs)
+        signomials, beforesubs, allsubs = self.signomials_et_al
+        beforesubs.signomials = signomials
```

Comment:
Ok, I still find it strange that the NomialData object doesn't already have all the info needed downstream, but I think that's outside the scope of this PR, and certainly lower priority than a lot of other issues.